File: bench/benches/fib.py
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", "lib"))
from bench import bench

sys.setrecursionlimit(100000)

def fib(n):
    return n if n < 2 else fib(n - 1) + fib(n - 2)

bench("fib", lambda: fib(30))

File: bench/benches/fib.go
package main

func fib(n int64) int64 {
	if n < 2 {
		return n
	}
	return fib(n-1) + fib(n-2)
}

func main() {
	bench("fib", func() int64 { return fib(30) })
}
